lib/puppet_x/puppetlabs/iis/powershell_manager.rb

```diff
@@ -169,7 +169,7 @@ def self.create_event(name, manual_reset = false, initial_state = false)
       WAIT_TIMEOUT = 0x00000102
       WAIT_FAILED = 0xFFFFFFFF
 
-      def self.wait_on(wait_object, timeout_ms = 50)
+      def self.wait_on(wait_object, timeout_ms = 1000)
```

Comment:
Having the message every 50ms just spams the hell of the output. I felt one message per second is a much better tradeoff.

This change can easily be removed from the PR.